view/frontend/web/template/payment/buckaroo_magento2_afterpay.html

```diff
@@ -165,15 +165,10 @@
                                    name="payment[buckaroo_magento2_afterpay][COCNumber]"
                                    class="input-text field"
                                    type="text"
+                                   data-validate="{required:true, minlength:8}"
```

Comment:
what would be the reason for this validation rule?

Reply:
@florinel-chis this is just a refactoring
change:
```
                              attr: {
                                            'data-validate': JSON.stringify({
                                                'required': true,
                                                'minlength': 8
                                            }),
                                        }
```
to:
`data-validate="{required:true, minlength:8}"`

because via attr does not work in safari browser on iphone